### PR TITLE
feat(frontend build and publish apps): add vnext step

### PR DESCRIPTION
**Pull request by kurtaqui**

### CODE REVIEW

File: .github/workflows/frontend-build-and-publish-apps.yml [hunk 1]
```diff
@@ -201,7 +201,7 @@ jobs:
         run: |
           git tag $APP_VERSION
           git push origin $APP_VERSION
-
+
```
Comment (tomaszbaron) [inline .github/workflows/frontend-build-and-publish-apps.yml:204]:
```suggestion

```

Comment (tomaszbaron) [reply .github/workflows/frontend-build-and-publish-apps.yml:204]: I meant inserting a new line without white spaces

File: .github/workflows/frontend-build-and-publish-apps.yml [hunk 2]
```diff
@@ -237,10 +237,29 @@ jobs:
...
       - name: Store npm artifact
         uses: actions/upload-artifact@v3
         if: failure()
         with:
           name: npm-logs
           path: ~/.npm/_logs/*
           retention-days: 2
+  merge:
+    name: merge
+    runs-on: ubuntu-latest
```
Comment (tomaszbaron) [inline .github/workflows/frontend-build-and-publish-apps.yml:250]: Don't we need define outputs to `build` job and add `needs: build` like in this example https://docs.github.com/en/actions/using-jobs/defining-outputs-for-jobs#example-defining-outputs-for-a-job ?

File: .github/workflows/frontend-build-and-publish-apps.yml [hunk 3]
```diff
@@ -237,10 +237,30 @@ jobs:
             fi
           fi
 
+
```
Comment (kurtaqui) [inline .github/workflows/frontend-build-and-publish-apps.yml:240]:
```suggestion
```

File: .github/workflows/frontend-build-and-publish-apps.yml [hunk 4]
```diff
@@ -244,3 +244,22 @@ jobs:
           name: npm-logs
           path: ~/.npm/_logs/*
           retention-days: 2
+  merge:
```
Comment (kurtaqui) [inline .github/workflows/frontend-build-and-publish-apps.yml:247]:
```suggestion

  merge:
```

**Review by stephenlautier — APPROVED** [review]

**Pull request merged**